configuration/module_test.go

```diff
@@ -38,19 +38,19 @@ func TestGetRealmConfigurations(t *testing.T) {
 	})
 	t.Run("Invalid JSON", func(t *testing.T) {
 		mockDB.EXPECT().QueryRow(gomock.Any(), realmID).Return(mockSQLRow)
-		mockSQLRow.EXPECT().Scan(gomock.Any()).DoAndReturn(func(ptrConfig *string, ptrAdminConfig *string) error {
-			*ptrConfig = `{`
-			*ptrAdminConfig = `{}`
+		mockSQLRow.EXPECT().Scan(gomock.Any()).DoAndReturn(func(dest ...interface{}) error {
+			*(dest[0]).(*string) = `{`
```

Comment:
It is strange to be obliged to loose the meaning of parameters provided to the function

Reply:
I did not manage to find a better way when updating to github.com/golang/mock v1.6.0. See : github.com/golang/mock v1.6.0